src/NexusMods.App.UI/Helpers/DoubleClickHelper.cs

```diff
@@ -0,0 +1,128 @@
+using System.Reactive;
+using System.Reactive.Disposables;
+using System.Reactive.Subjects;
+using Avalonia.Controls;
+using Avalonia.Input;
+using Avalonia.Interactivity;
+using JetBrains.Annotations;
+
+namespace NexusMods.App.UI.Helpers;
+
+[PublicAPI]
+public static class DoubleClickHelper
+{
+    // NOTE(erri120): Windows uses 500ms (https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-setdoubleclicktime?redirectedfrom=MSDN#parameters)
+    // This can be fine-tuned later.
+    public static readonly TimeSpan DefaultTimeout = TimeSpan.FromMilliseconds(500);
+
+    /// <summary>
+    /// Registers a double click handler and returns an observable that triggers whenever
+    /// a double click occurs.
+    /// </summary>
+    public static IObservable<Unit> AddDoubleClickHandler(
+        this Control control,
+        RoutingStrategies routingStrategy = RoutingStrategies.Bubble,
+        bool ignoreHandledEvents = false)
```

Comment:
I would have this set to true by default as that is the default behavior for handlers

Reply:
I'd rather have this default to `false` so it works 99% of the time, instead of setting it to `true` for it works only 30% of the time.

Reply:
I expect this to work 80% of the time or more, unless avalonia is really crap, but I haven't tried. I guess you speak from experience?